akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/ActorContext.scala

```diff
@@ -279,4 +280,10 @@ trait ActorContext[T] extends akka.actor.typed.ActorContext[T] {
     createRequest:   java.util.function.Function[ActorRef[Res], Req],
     applyToResponse: BiFunction[Res, Throwable, T]): Unit
 
+  /**
+   * Applies the provided function when the CompletionStage is completed (see `CompletionStage`)
+   * and sends the result as a message to this Actor (“`self`”).
+   */
```

Comment:
add
```
    * This method is thread-safe and can be called from other threads than the ordinary
    * actor message processing thread, such as [[java.util.concurrent.CompletionStage]] callbacks.
```

Reply:
I'd like it to first explain how it is useful rather than what it does, 

something like "Makes it possible to adapt the completed or failed value of the completion stage into a message that the actor understands by applying a transformation function"